pkg/source/imgpkg.go

```diff
@@ -36,7 +39,7 @@ type RegistryOpts struct {
 	RegistryToken    string
 }
 
-func ImgpkgPush(ctx context.Context, dir string, excludedFiles []string, registryOpts *RegistryOpts, image string) (string, error) {
+func ImgpkgPush(ctx context.Context, dir string, excludedFiles []string, registryOpts *RegistryOpts, image string, c *cli.Config) (string, error) {
```

Comment:
imgpkg is implementation and CLI config shouldnt be leaked into func def

Reply:
Modified the file to not to send cli config to imgpkg